app/src/main/java/com/dragic/gamehunter/utils/NumberFormattingUtils.kt

```diff
@@ -0,0 +1,27 @@
+package com.dragic.gamehunter.utils
+
+import android.icu.number.Notation
+import android.icu.number.NumberFormatter
+import android.icu.number.Precision
+import android.icu.util.Currency
+import android.icu.util.MeasureUnit
+import android.icu.util.ULocale
+import android.os.Build
+import androidx.annotation.RequiresApi
+
+@RequiresApi(Build.VERSION_CODES.R)
```

Comment:
Oh, probably because of NumberFormatter? You can just make min SDK 30 in the project and remove these. Doesn't matter all that much for this sample project 😄